.circleci/config.yml

```diff
@@ -371,9 +371,10 @@ jobs:
       - run:
           name: "Install Adapter Dependencies"
           command: |
-            mkdir ~/adapter-deps ~/adapter-deps/install
+            mkdir ~/adapter-deps ~/adapter-deps/install ~/adapter-deps/google-cloud-cpp-installed/
```

Comment:
If this ` ~/adapter-deps/google-cloud-cpp-installed/` is mandatory directory than this should be done inside install script not in the CiCd.

Reply:
Good catch, extra directory removed.